sprocs/workspaces_state_update.sql

```diff
@@ -0,0 +1,15 @@
+CREATE OR REPLACE FUNCTION
+    workspaces_state_update(
+        workspace_id bigint,
+        workspace_state text
+    ) returns void
+AS $$
+BEGIN
+    UPDATE
+        workspaces as w
+    SET
+        state = workspace_state::enum_workspace_state
+    WHERE
+        w.id = workspace_id;
+END;
+$$ LANGUAGE plpgsql VOLATILE;
```

Comment:
Should we be writing to `workspace_logs` here in this sproc?

Reply:
Good idea!

Reply:
Added in c51024b